doc/pipeline-01.yaml

```diff
@@ -0,0 +1,63 @@
+name: mlflow-sklearn-e2e
+description: |
+  End-to-end pipeline template that takes in an MLFlow compatible codeset,
+  runs the MLFlow project to train a model, then creates a KFServing prediction
+  service that can be used to run predictions against the model."
+inputs:
+  - name: mlflow-codeset
+    description: an MLFlow compatible codeset
+    type: codeset
+  - name: predictor
+    description: type of predictor engine
+    type: string
+    default: auto
+outputs:
+  - name: prediction-url
+    description: "The URL where the exposed prediction service endpoint can be contacted to run predictions."
+    type: string
+steps:
+  - name: builder
+    image: ghcr.io/fuseml/mlflow-builder:1.0
+    inputs:
+      - codeset:
+          name: '{{ inputs.mlflow-codeset }}'
+          path: /project
+    outputs:
+      - name: mlflow-env
+        image:
+          dockerfile: '/project/.fuseml/Dockerfile'
+          name: 'registry.fuseml-registry/mlflow-builder/{{ inputs.mlflow-codeset.name }}:{{ inputs.mlflow-codeset.version }}'
+  - name: trainer
+    image: '{{ steps.builder.outputs.mlflow-env }}'
+    inputs:
+      - codeset:
+          name: '{{ inputs.mlflow-codeset }}'
+          path: '/project'
+    outputs:
+      - name: mlflow-model-url
+    env:
+      - name: MLFLOW_TRACKING_URI
+        value: "http://mlflow"
+      - name: MLFLOW_S3_ENDPOINT_URL
+        value: "http://mlflow-minio:9000"
+      - name: AWS_ACCESS_KEY_ID
+        value: CdkokHmIRZZ4s8PAQ1RI
+      - name: AWS_SECRET_ACCESS_KEY
+        value: LyXpfxpJ58B0nkQEopknq0h3lwV1jivv2MlWFM8R
+  - name: predictor
+    image: 'ghcr.io/fuseml/kfserving-predictor:v0.1'
+    inputs:
+      - name: model
+        value: '{{ steps.trainer.outputs.mlflow-model-url }}'
+      - name: predictor
+        value: '{{ inputs.predictor }}'
+      - codeset:
+          name: '{{ inputs.mlflow-codeset }}'
+          path: '/project'
```

Comment:
I can see that the `path` option gets repeated here and there. Couldn't it have default value?

Reply:
Maybe, do you mind proposing that in https://github.com/fuseml/wiki/pull/1 ?